devtools-integration-tests/src/test/java/org/ballerina/devtools/cmd/TestUtils.java

```diff
@@ -0,0 +1,205 @@
+     *
+     * @param distributionZipPath Path to the distribution.
+     * @throws ZipException Error occurred when extracting.
+     */
+    public static void prepareDistribution(Path distributionZipPath) throws ZipException {
+        OUT.println("Extracting: " + distributionZipPath.normalize());
+        ZipFile zipFile = new ZipFile(distributionZipPath.toFile());
+        zipFile.extractAll(TEST_DISTRIBUTION_PATH.toAbsolutePath().toString());
+    }
+
+    /**
+     * Delete the temporary directory used to extract distributions.
+     *
+     * @throws IOException If temporary directory does not exists.
+     */
+    public static void cleanDistribution() throws IOException {
+        FileUtils.deleteDirectory(TEST_DISTRIBUTION_PATH.toFile());
+    }
+
+    /**
+     * Create Settings.toml inside the home repository.
+     *
+     * @throws IOException i/o exception when writing to file
+     */
+    static void createSettingToml(Path dirPath) throws IOException {
+        String content = "[central]\n accesstoken = \"" + getToken() + "\"";
+        Files.write(dirPath.resolve("Settings.toml"), content.getBytes(), StandardOpenOption.CREATE,
+                StandardOpenOption.TRUNCATE_EXISTING);
+    }
+
+    /**
+     * Get token of ballerina-central-bot required to push the module.
+     *
+     * @return token required to push the module.
+     */
+    private static String getToken() {
+        // staging and dev both has the same access token
+        return System.getenv("devCentralToken");
+    }
+
+    /**
+     * Get environment variables and add ballerina_home as a env variable the tmp directory.
+     *
+     * @return env directory variable array
+     */
+    static Map<String, String> getEnvVariables() {
+        Map<String, String> envVarMap = System.getenv();
+        Map<String, String> retMap = new HashMap<>();
+        envVarMap.forEach(retMap::put);
+        return retMap;
+    }
+
+    /**
+     * Delete files inside directories.
+     *
+     * @param dirPath directory path
+     * @throws IOException throw an exception if an issue occurs
+     */
+    static void deleteFiles(Path dirPath) throws IOException {
+        if (dirPath == null) {
+            return;
+        }
+        Files.walk(dirPath).sorted(Comparator.reverseOrder()).forEach(path -> {
+            try {
+                Files.delete(path);
+            } catch (IOException e) {
+                Assert.fail(e.getMessage(), e);
+            }
+        });
+    }
+
+    /**
+     * Get generate bala log.
+     *
+     * @param org      org name
+     * @param pkgName  package name
+     * @param platform platform name
+     * @param version  package version
+     * @return generate bala log
+     */
+    static String getGenerateBalaLog(String org, String pkgName, String platform, String version) {
+        return "Creating bala\n" + "\ttarget/bala/" + org + "-" + pkgName + "-" + platform + "-" + version + ".bala";
```

Comment:
This function and `getBalaPath` are not used anywhere.

```suggestion
        return "Creating bala\n" + "\target/bala/" + org + "-" + pkgName + "-" + platform + "-" + version + ".bala";
```